### PR TITLE
Add troubleshooting section to running unit test section

### PR DESCRIPTION
**Pull request by nielslange**
### Changes proposed in this Pull Request:

Following the steps to run unit test, might lead to a problem that was firstly mentioned by Daniel Bachhuber in 2015 in https://github.com/wp-cli/wp-cli/issues/1938#issuecomment-124237542. This PR adds troubleshooting instructions in case a unit test cannot be executed and an error message appears instead.

### How to test the changes in this Pull Request:

1. Check out this PR.
2. Look up the file `tests/README.md`.
3. Check the section `### Troubleshooting`.

---

@Konamiman I added you as a reviewer and noticed that my linting settings changed a few additional parts in the file. Please let me know if I shall update my PR and remove the additional markdown lintings.

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -66,17 +67,34 @@ A text code coverage summary can be displayed using the `--coverage-text` option
...
 
+### Troubleshooting
+
+In case you're unable to run the unit tests, you might see an error message similar to:
+
+```
+Fatal error: require_once(): Failed opening required '/var/folders/qr/3cnz_5_j3j1cljph_246ty1h0000gn/T/wordpress-tests-lib/includes/functions.php' (include_path='.:/usr/local/Cellar/php@7.4/7.4.23/share/php@7.4/pear') in /Users/nielslange/Plugins/woocommerce/tests/legacy/bootstrap.php on line 59
+```
+
+If you run into this problem, simply delete the WordPress test directory and run teh installer again. In this particular case, you'd run the following command:
```
Comment (Konamiman) [inline tests/README.md]: Typo: `teh`

**Review by Konamiman — CHANGES_REQUESTED** [review]
Hi @nielslange, yes, I think it's better to undo the changes done by the linter, they aren't really needed and they are also breaking the doc structure (e.g. the second list in "Writing tests"). Also there are merge conflicts for the file, could you please address these too?

File: tests/README.md (hunk 1 above)
Comment (nielslange) [reply tests/README.md]: Oh, my no. 1 typo. 🙈 Just fixed it.

**Comment by nielslange** [comment]
> Hi @nielslange, yes, I think it's better to undo the changes done by the linter, they aren't really needed and they are also breaking the doc structure (e.g. the second list in "Writing tests"). Also there are merge conflicts for the file, could you please address these too?

Thanks for your review, @Konamiman. I created a fresh PR that only contains the newly added section and does not apply any linting changes.

File: plugins/woocommerce/tests/README.md [hunk 2]
```diff
@@ -66,6 +66,22 @@ A text code coverage summary can be displayed using the `--coverage-text` option
...
+Fatal error: require_once(): Failed opening required '/var/folders/qr/3cnz_5_j3j1cljph_246ty1h0000gn/T/wordpress-tests-lib/includes/functions.php' (include_path='.:/usr/local/Cellar/php@7.4/7.4.23/share/php@7.4/pear') in /Users/nielslange/Plugins/woocommerce/tests/legacy/bootstrap.php on line 59
+```
+
+If you run into this problem, simply delete the WordPress test directory and run the installer again. In this particular case, you'd run the following command:
+
+```
+$ rm -rf /var/folders/qr/3cnz_5_j3j1cljph_246ty1h0000gn/T/wordpress-tests-lib
+$ tests/bin/install.sh woocommerce_tests_1 root root
+# Note that woocommerce_tests changed to woocommerce_tests_1 as the database woocommerce_tests already exists due to the prior command.
+```
```
Comment (Konamiman) [inline plugins/woocommerce/tests/README.md]: Just a small thing: the note about the database name could go outside the code block for better visibility:

```
$ rm -rf /var/folders/qr/3cnz_5_j3j1cljph_246ty1h0000gn/T/wordpress-tests-lib
$ tests/bin/install.sh woocommerce_tests_1 root root
```
(Note that `woocommerce_tests` changed to `woocommerce_tests_1` as the database `woocommerce_tests` already exists due to the prior command)

Comment (nielslange) [reply plugins/woocommerce/tests/README.md]: As suggested, I placed the note outside the code block. I haven't placed it in brackets, though. Please let me know if you prefer to place it in brackets, as you suggested initially. I opted against brackets, as personally don't see the added value in having them. That said, if you prefer having them, just let me know. 😉

**Comment by Konamiman** [comment]
Looks good. "Troubleshooting" was missing from the table of contents but I have added it myself.

**Review by Konamiman — APPROVED** [review]

**Pull request merged**